docker-compose.yaml

```
@@ -1,10 +1,22 @@
+      - ./test_ollama.py:/src/main.py
+    env_file:
+      - .env
+  simplemind-test:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - ./simplemind:/src/simplemind
+      - ./test_ollama.py:/src/main.py
     env_file:
       - .env
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)